test/functionalTest/cases/719_servicepath_for_registrations/cm_forwards_cb_with_servicePath.test

```diff
@@ -0,0 +1,456 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
* Step 7 will resolve to E4 and E5 in /c2/c3/e4. Not sure if this functionality is implemented, as it result in more than one registration matching in the scope of the same queryContext.
* Step 8 will resolve to E1 (in /c1/c1/e1) and E2 (/c1/c2/e2). Similar case than the one above.
* Step 9 will resolve to several cases also.

I think that in all the cases, only one forwarding result should happen.